### ISSUE TITLE
Support render_as options from the UV metadata panel

### ISSUE THREAD

**Issue by ShanaLMoore** [notch8/utk-hyku#299]
In LV, a user is able to click on facetable links from the parent's metadata side panel. 

In Essi, the user cannot. @crisr15 Should this be supported in iiif_print?

**Comment by ShanaLMoore** [comment]
As discussed in Crystal's Office hr, this is not a concern at the moment. We will re visit this once the main iiif_print work is complete.

**Comment by jillpe** [comment]
closed, this is no longer relevant
